src/extension/iSamples_textSearch.js

```diff
@@ -49,9 +49,22 @@ class TextSearch extends React.Component {
   }
 
   render() {
-    const { label, bootstrapCss, collapse } = this.props;
+    const { label, bootstrapCss, collapse, type } = this.props;
 
-    return (
+    return (type === "non-facet" ?
+      // reserved if we need to render label for non type
+      // <li className={cx({ "list-group-item": bootstrapCss })}>
+      //   <header>
+      //     <h5>
+      //       {bootstrapCss ? (<span>
+      //         <span className={cx("glyphicon glyphicon-check")} />{" "}
+      //       </span>) : null}
+      //       {label}
+      //     </h5>
+      //   </header>
+      // </li>
```

Comment:
reserved if we want to render the `non-facet` type fields' labels in the left pane.